evap/staff/importers.py

```diff
@@ -390,9 +391,26 @@ def process(cls, excel_content, semester, vote_start_datetime, vote_end_date, te
 
 
 class UserImporter(ExcelImporter):
-    def read_one_user(self, data):
+
+    def __init__(self):
+        super().__init__()
+        self._read_user_data_hashes = dict()
```

Comment:
This is used to report on duplicated lines. @karyon preferred to warn on duplicate lines and not already on duplicate emails. 
We might have preferred to report on duplicate lines in the ExcelImporter Superclass, but that would be a bigger rewrite, I thought.